pages/common/hadolint.md

```diff
@@ -0,0 +1,24 @@
+# hadolint
+
+> Dockerfile linter.
+> More information: <https://github.com/hadolint/hadolint>.
+
+- Lint a Dockerfile:
+
+`hadolint {{path/to/Dockerfile}}`
+
+- Lint a Dockerfile outputting in json:
+
+`hadolint --format {{json}} {{path/to/Dockerfile}}`
+
+- Lint a Dockerfile outputting in a codeclimate-friendly format:
+
+`hadolint --format {{codeclimate}} {{path/to/Dockerfile}}`
```

Comment:
I don't know if `codeclimate` is widely used, but I prefer a more generic example to display all the possible formats in the tldr pages:
```suggestion
`hadolint --format {{tty|json|checkstyle|codeclimate|codacy}} {{path/to/Dockerfile}}`
```

Reply:
Yeah. This is what I was looking for. This is a much better output. Thanks